var/spack/repos/builtin/packages/helics/package.py

```diff
@@ -17,8 +17,10 @@ class Helics(CMakePackage):
 
     maintainers = ['nightlark']
 
+    version('bms', branch='install-benchmarks', submodules=True)
```

Comment:
Note that Spack will treat this version as being older than all other versions.

Reply:
Good catch. Now that v2.6.0 has the benchmarking changes, that branch is no longer needed.